CommandLine/protos/resources/Font.proto

```diff
@@ -0,0 +1,40 @@
+syntax = "proto2";
+package buffers.resources;
+
+import "options.proto";
+
+message Font {
+  message Glyph {
+    optional uint32 character = 1; 
+    optional uint32 x = 2;
+    optional uint32 y = 3; 
+    optional uint32 width = 4 [(gmx) = "w"];
+    optional uint32 height = 5 [(gmx) = "h"]; 
+    optional int32 shift = 6; 
+    optional int32 offset = 7;
+  }
+
+  message Range {
+    optional int32 min = 1 [(gmx) = "GMX_SPLIT/0"];
+    optional int32 max = 2 [(gmx) = "GMX_SPLIT/1"];
+  }
+
+  optional string name = 1;
+  optional int32 id = 2 [(gmx) = "GMX_DEPRECATED"];
+
+  optional string font_name = 3 [(gmx) = "name"];
+  optional int32 size = 4;
+  optional bool bold = 5;
+  optional bool italic = 6;
+
+  repeated Range ranges = 7 [(gmx) = "ranges/range0"];
+  repeated Glyph glyphs = 8 [(gmx) = "glyphs/glyph"];
+  
+  optional bool renderhq = 10;
+  optional uint32 charset = 11;
+  optional uint32 aa = 12;
+  optional bool include_ttf = 13 [(gmx) = "includeTTF"];
+  optional string ttf_file = 14 [(gmx) = "TTFName"]; 
+  optional int32 texture_group = 15 [(gmx) = "texgroups/texgroup0"];

```

Comment:
This is a texture atlas, I assume..? I didn't realize GM was quite this far along.

Reply:
I'm not sure what it is. I've never seen anything other than texture0 in a gmx but I added it because I wanted to cover all the gmx fields

Reply:
Yes but @JoshDreamland do not be fooled by the number 0, there is only ever one, so our use of the gmx option here is safe. Textures cannot be added to more than one texture group, at least in GMS 1.4.